scripts/qa_crowbarsetup.sh

```diff
@@ -1525,7 +1525,7 @@ EOF
         /opt/dell/bin/json-edit -a attributes.network.networks.public.ranges.host.end -v 10.162.211.191 $netfile
     fi
     # Setup network attributes for custom MTU
-    if [[ $want_mtu_size -ne 1500 ]]; then
+    if [[ $want_mtu_size ]] && [[ $want_mtu_size -ne 1500 ]]; then
```

Comment:
Changed scripts/lib/mkcloud-common.sh to set a default value for $want_mtu_size.
Just to be sure, I would favour to keep the check here to ensure that the same error we have encountered does not occur again when the default value is changed/refactored.